ext/ManifoldsTestExt/tests_group.jl

```diff
@@ -463,11 +463,16 @@ function test_group(
     end
 
     test_lie_bracket && Test.@testset "Lie bracket" begin
-        # anticommutativity
         X = Xe_pts[1]
         Y = Xe_pts[2]
+        Z = Xe_pts[3]
```

Comment:
It's just that two `Xe_pts` instead of three were given. I thought there should be three, as for `g_pts` there should be three. (`Tests general properties of the group G, given at least three different points`). My local testing did not catch this... I'll investigate.

Reply:
Ah I was already worried you'd access some components of a vector, but right it should be the first 3 points. Well I hope we have 3 most of the time.

Reply:
There are indeed several cases where two are given, not three, which causes the errors.

Reply:
If you see a good way to give them a different third (different from the other 2) that would probably be the best fix.
I was not aware we had that, but maybe until now also no one needed 3 points on a group in a test.

Reply:
I agree, that's the best solution. I think there is always three points, but sometimes only two vectors. I'll add a third vector to all tests that need one, that should address the issue.